Sources/PointFree/Pricing.swift

```diff
@@ -222,20 +247,56 @@ private let teamPricingRowView = View<Pricing> { pricing in
         h6([`class`([Class.pf.type.title6, Class.pf.colors.fg.purple])], ["Yearly Plan"]),
         p([`class`([Class.pf.colors.fg.purple])], ["How many in your team?"]),
         input([
-          type(.number),
-          min(Pricing.validTeamQuantities.lowerBound),
+          `class`([numberSpinner, Class.pf.colors.fg.purple]),
           max(Pricing.validTeamQuantities.upperBound),
+          min(Pricing.validTeamQuantities.lowerBound),
           name("pricing[team]"),
+          onchange(
```

Comment:
A lil inline JavaScript never hurt anyone.

Reply:
lol never once hurt ANYONE

whatdya think about moving it to a separate var outside the view so that it's easy to digest? also the `\` arent necessary. turns out JS in inline code can have newlines! I do that for the video player on the episode page